apps/server/src/modules/user-login-migration/service/school-migration.service.ts

```diff
@@ -63,6 +64,18 @@ export class SchoolMigrationService {
 		return existingSchool;
 	}
 
+	async completeMigration(schoolId: string): Promise<void> {
+		const notMigratedUsers: Page<UserDO> = await this.userService.findUsers({ schoolId, isOutdated: false });
```

Comment:
Yes, its filter all users with no outdatedSince (where outdatedSince is not defined / undefined), this does not include migrated users, because they will not have a outdatedSince set. Lets talk

Reply:
Okay, added previousExternalId to query. Done :)